arrowhead-clients/provider.py

```diff
@@ -0,0 +1,56 @@
+import sys
+from datetime import datetime
+sys.path.insert(1, '/home/ruben/git/client-library-python')
+import arrowhead_client.api as ar
+
+provider_app = ar.ArrowheadHttpClient(
+        system_name='lcm_provider',
+        address='127.0.0.1',
+        port=5000,
```

Comment:
This is the IP for the provider. If it is not running on the same machine it should be changed to the IP on the machine it is running on. This IP and port are registered in the arrowhead core systems database when the provider starts. When the consumer starts, it will look for the provider IP in the arrowhead core systems database and connect to it.